apps/api/src/api.module.ts

```diff
@@ -128,7 +125,7 @@ import * as QueueConstants from '../../../libs/common/src/utils/queues';
     ScheduleModule.forRoot(),
   ],
   providers: [ApiService, GraphStateManager, ConfigService],
-  controllers: [GraphController, HealthController],
+  controllers: [GraphControllerV1, HealthController],
```

Comment:
Nit: Is it necessary to alter the filename and name of the controller? I've just been changing the path